### PR TITLE
Add Harvest Forecast Guide to DVELP Cookbook

### PR DESCRIPTION
**Pull request by latimerjason**
Add Harvest Forecast guide and update read me with link

DVELP uses Forecast to plan our time and estimate availability before
taking on new work. Share the process of how DVELP plans resources in
our GitHub cookbook

### CODE REVIEW

File: guides/harvest-forecast.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+We take a bottom up approach to planning at DVELP. We trust in the team's expertise to estimate how much time they need for the work on hand.
+
+In the **Projects** view, Team members are added to their active projects for their expected hours effort per day needed. We plan in support for active projects against that project directly, particularly for the critical time after launch.  We also have team members planned for general ongoing support to catch the odd help request from our long term clients. Public holidays and vacations are accounted for, as are internal R&D projects. 
+
+We target having the team planned at 80% of capacity to allow some contingency in case of illness or last minute requests. 
+
+When the dust settles, we look at the **Team** view to see if anyone has too much or too little on their plate and work with everyone to share the load. 
+
+##Planning Flow 
+New requests and changes are planned into Forecast daily. Any changes to the current week are reviewed with the impacted teams in real time.
```
Comment (latimerjason) [inline guides/harvest-forecast.md:17]: New client requests and changes are planned into Forecast daily.

File: guides/harvest-forecast.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+In the **Projects** view, Team members are added to their active projects for their expected hours effort per day needed. We plan in support for active projects against that project directly, particularly for the critical time after launch.  We also have team members planned for general ongoing support to catch the odd help request from our long term clients. Public holidays and vacations are accounted for, as are internal R&D projects. 
+
+We target having the team planned at 80% of capacity to allow some contingency in case of illness or last minute requests. 
+
+When the dust settles, we look at the **Team** view to see if anyone has too much or too little on their plate and work with everyone to share the load. 
+
+##Planning Flow 
+New requests and changes are planned into Forecast daily. Any changes to the current week are reviewed with the impacted teams in real time. 
+
+Full projects are estimated and planned in their entierty. Small improvement works (<2 days) are planned in detail across the next two weeks.
```
Comment (tom-mullen) [inline guides/harvest-forecast.md]: @latimerjason typo 'entirety'

**Review by tom-mullen — CHANGES_REQUESTED** [review]
Just one typo

**Comment by latimerjason** [comment]
@tom-mullen - change has been made. Can we ship?

**Review by tom-mullen — APPROVED** [review]

**Pull request merged**